tests/LinAlg/matrixTestsSparse.hpp

```diff
@@ -432,10 +500,12 @@ class MatrixTestsSparse : public TestBase
   virtual real_type getLocalElement(const hiop::hiopMatrix* a, local_ordinal_type i, local_ordinal_type j) = 0;
   virtual real_type getLocalElement(const hiop::hiopVector* x, local_ordinal_type i) = 0;
   virtual real_type* getMatrixData(hiop::hiopMatrixSparse* a) = 0;
+  virtual real_type getMatrixData(hiop::hiopMatrixSparse* a, local_ordinal_type i, local_ordinal_type j) = 0;
   virtual const local_ordinal_type* getRowIndices(const hiop::hiopMatrixSparse* a) = 0;
   virtual const local_ordinal_type* getColumnIndices(const hiop::hiopMatrixSparse* a) = 0;
   virtual local_ordinal_type getLocalSize(const hiop::hiopVector* x) = 0;
   virtual int verifyAnswer(hiop::hiopMatrix* A, real_type answer) = 0;
+  virtual int verifyAnswer(hiop::hiopMatrix* A, local_ordinal_type nnz_st, local_ordinal_type nnz_ed, const double answer) = 0;
```

Comment:
@ashermancinelli Please verify. I add one function to verify the answers for a sparse matrix, since the other "verifyAnswer" requires a dense input matrix "A". See file matrixTestsSparseTriplet.cpp for examples. NOTE that since this branch is developed based on 'master', it doesn't has the RAJA implementation.

Reply:
I think if you're just checking a single element, probably better to use `getLocalElement` and check result